openquake/calculators/base.py

```diff
@@ -67,6 +69,38 @@
                         ('len', U16)])
 
 
+def remove_heading_2_words(s):
+    return ' '.join(s.split(' ')[2:]) if isinstance(s, str) else s
+
+
+def get_aelo_changelog():
+    dic = collections.defaultdict(list)
+    c = configparser.ConfigParser()
+    changelog_path = os.path.join(
+        config.directory.mosaic_dir, 'aelo_changelog.ini')
+    c.read(changelog_path)
+    for sec in c.sections():
+        dic['AELO VERSION'].append(sec)
+        for k, v in c.items(sec):
+            dic[k].append(v)
+    df = pandas.DataFrame(dic)
+    df = df.drop(columns=['private'])
+    # NOTE: extracting version from a string like "AELO VERSION 1.0.0"
+    df['AELO VERSION'] = df['AELO VERSION'].apply(remove_heading_2_words)
+    df = df[~df['AELO VERSION'].str.startswith('_')]
```

Comment:
I would remove this feature, i.e. change _1.1.0 -> 1.1.0

Reply:
Done